src/app/scheduler/components/SchedulerCalendar.tsx

```diff
@@ -0,0 +1,155 @@
+import dayjs from 'dayjs';
+import customParseFormat from 'dayjs/plugin/customParseFormat';
+import utc from 'dayjs/plugin/utc';
+import moment from 'moment';
+import { MutableRefObject, useMemo, useRef } from 'react';
+import 'react-big-calendar/lib/sass/styles.scss';
+import { Calendar, momentLocalizer, Event, EventProps } from 'react-big-calendar';
+import { RRule } from 'rrule';
+
+import { CalendarEvent } from './CalendarEvent';
+
+dayjs.extend(utc);
+dayjs.extend(customParseFormat);
+
+const localizer = momentLocalizer(moment);
+
+const eventStyleGetter = ({ resource }: Event) => {
+  var style = {
+    backgroundColor: resource && resource.color,
+    borderRadius: '0px',
+    opacity: 0.8,
+    color: 'black',
+    border: '0px',
+    display: 'block',
+  };
+  return {
+    style: style,
+  };
+};
+
+const CustomEvent = ({ title, event }: EventProps) => {
+  return (
+    <span>
+      {title}, {event.resource && event.resource.sectionCode}, {event.resource && event.resource.location}
+    </span>
+  );
+};
+
+export interface SchedulerCalendarProps {
+  /**
+   * CalendarEvents
+   * Parses events that can go into the calendar from this
+   */
+  calendarEvents?: CalendarEvent[];
+}
+
+export function SchedulerCalendar({ calendarEvents }: SchedulerCalendarProps): JSX.Element {
+  const minEventDate: MutableRefObject<Date | undefined> = useRef(undefined);
+
+  const computeMeetingTimeDays = (calendarEvent: CalendarEvent) => {
+    const days = calendarEvent.meetingTime.days;
+    const daysRRule = [];
+
+    if (days.includes('M')) {
+      daysRRule.push(RRule.MO);
+    }
+    if (days.includes('T')) {
+      daysRRule.push(RRule.TU);
+    }
+    if (days.includes('W')) {
+      daysRRule.push(RRule.WE);
+    }
+    if (days.includes('R')) {
+      daysRRule.push(RRule.TH);
+    }
+    if (days.includes('F')) {
+      daysRRule.push(RRule.FR);
+    }
+
+    return daysRRule;
+  };
+
+  const events = useMemo(() => {
+    const events: Event[] = [];
+    calendarEvents?.forEach((calendarEvent) => {
+      try {
+        if (calendarEvent.meetingTime.time === 'TBA') {
```

Comment:
I think a `indexOf` might be better so if there's some weird formatting it wouldn't die from it.